Src/ILGPU.Algorithms.Tests.CPU/ILGPU.Algorithms.Tests.CPU.csproj

```diff
@@ -0,0 +1,53 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

Comment:
ILGPU currently uses netcoreapp2.1 for all test cases due to some issues in the test SDK. Have you experienced any problems?

Reply:
I'll update the patch to use netcoreapp2.1 and the latest nuget packages.

No, I have not experienced any issues.

Reply:
Awesome, thanks :)